### ISSUE TITLE
About MonoTAKD Student Model Performance

### ISSUE THREAD

**Issue by Min100KM** [hoiliu-0801/MonoTAKD#4]
I conducted experiments to reproduce the results of MonoTAKD, and i got some weird high performance results.

I used the student-trained weight file, `monotakd_kitti_student-8015e10.pth` , provided in your GitHub repository.

`python test_TAKD.py --cfg cfgs/kitti_models/TAKD/TAKD-scd/kitti_R50_scd_TAKD.yaml --ckpt ../checkpoints/monotakd_kitti_student-8015e10.pth`

The results below I obtained were too high. Could you please check If there is any issue with the provided weight file?

![Image](https://github.com/user-attachments/assets/ce495665-0042-4e82-beac-8c7f2e821721)

Thank you so much.

**Comment by hoiliu-0801** [comment]
Since our model was pre-trained on other autonomous driving datasets similar to KITTI raw, the results may exhibit overfitting on the KITTI validation set. You should verify your final results using the KITTI official evaluation.
